dev-tools/vscode-metatype-support/src/extension.ts

```diff
@@ -0,0 +1,52 @@
+import * as path from "path";
+import { ExtensionContext, workspace } from "vscode";
+
+import {
+  LanguageClient,
+  LanguageClientOptions,
+  ServerOptions,
+  TransportKind,
+} from "vscode-languageclient/node";
+
+let client: LanguageClient;
+
+export function activate(context: ExtensionContext) {
+  const tsServerDir = context.asAbsolutePath(
+    path.join("..", "ts-language-server"),
+  );
+  const serverOptions: ServerOptions = {
+    run: {
+      command: "typegraph-ts-server",
+      args: ["--stdio"],
+      transport: TransportKind.stdio,
+    },
+    debug: {
+      command: "deno",
```

Comment:
question: Any reason you decided to write the lsp in deno? now that `meta-cli` embeds its own deno, the deno binary itself will be a unique requirement of the lang server. If the vs-code environment provides node by default, might be better in that sense.